open-company/open-company-web
# Company loading

card: https://trello.com/c/orcBRl0X

to test:
- deploy this (there is an issue currently on mainline while deploying but i'll fix it soon)
- check the company list
- select a company
- go back to the company list
- select another company
- repeat as long as you like
- if you never see the company not found or companies not found message you can merge
